styles/terminal.less

```diff
@@ -70,7 +70,5 @@
 @import (less) "../node_modules/xterm/css/xterm.css";
 
 atomic-terminal {
-  .terminal.xterm {
-    padding: 1rem;
-  }
+  padding: @component-icon-padding;
```

Comment:
```suggestion
  padding: @component-padding;
```

Reply:
To make it consistent with tree-view I think we should do icon padding.